net/haproxy/Makefile

```diff
@@ -1,7 +1,7 @@
 PLUGIN_NAME=		haproxy
-PLUGIN_VERSION=		3.12
+PLUGIN_VERSION=		4.0
 PLUGIN_COMMENT=		Reliable, high performance TCP/HTTP load balancer
-PLUGIN_DEPENDS=		haproxy24
+PLUGIN_DEPENDS=		haproxy
```

Comment:
I suppose we need the tools.git support for that? :)

Reply:
PR https://github.com/opnsense/tools/pull/332 :)

Reply:
all done, thanks!